include/dosbox.h

```diff
@@ -36,8 +36,11 @@ extern bool shutdown_requested;
 [[noreturn]] void E_Exit(const char *message, ...)
         GCC_ATTRIBUTE(__format__(__printf__, 1, 2));
 
-void MSG_Add(const char*,const char*); //add messages to the internal languagefile
-const char* MSG_Get(char const *);     //get messages from the internal languagefile
+// add messages to the internal languagefile
+void MSG_Add(const char*, const char*);
+void MSG_Add(const char*, const char*, bool is_utf8);
```

Comment:
Well, let's see how a call of the UTF-8 variant would look like:

```
MSG_Add(foo, bar, true);
```

Huh, what does that `true` signify? That doesn't read well at all, and I think booleans should be only used as function arguments when you can infer the meaning of the boolean from the name of the function (and even then, a single boolean argument max).

I strongly suggest to get rid of that boolean flag and introduce a new function called `MSG_Add_Utf8`.

Reply:
got rid of the extended function. It is actually not needed outside of messages.cpp, and I assume the plan is to migrate to gettext nevertheless.